plume-cli/src/main.rs

```diff
@@ -27,6 +27,7 @@ fn main() {
 
     dotenv::dotenv().ok();
     let conn = Conn::establish(CONFIG.database_url.as_str());
+    let _ = conn.as_ref().map(|conn| Instance::cache_local(conn));
```

Comment:
Do you really need this `let` here?

Reply:
just to explicitly ignore a warning about unused result